projects/kalidea/kaligraphi/src/lib/02-form/kal-select-virtual-scroll/kal-select-virtual-scroll.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="kal-select-virtual-scroll__trigger" [id]="id">
+  <div (click)="open()" (focus)="open()">
```

Comment:
When clicking on the input it will trigger 2 times your open function

Reply:
It is necessary if we want to use the focus, same as the other KalSelect, a condition is in the open function to reject the second call